[2023-04-08T10:16:36.959+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T14:20:00+00:00 [queued]>
[2023-04-08T10:16:37.123+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T14:20:00+00:00 [queued]>
[2023-04-08T10:16:37.125+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:37.126+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-08T10:16:37.128+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:37.353+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-06 14:20:00+00:00
[2023-04-08T10:16:37.442+0000] {standard_task_runner.py:55} INFO - Started process 13675 to run task
[2023-04-08T10:16:37.502+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-06T14:20:00+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmp7r4mq4_g']
[2023-04-08T10:16:37.529+0000] {standard_task_runner.py:83} INFO - Job 306: Subtask respond
[2023-04-08T10:16:38.371+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-06T14:20:00+00:00 [running]> on host cfece00ff772
[2023-04-08T10:16:38.981+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T14:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-06T14:20:00+00:00
[2023-04-08T10:16:38.989+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-08T10:16:39.122+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230406T142000, start_date=20230408T101637, end_date=20230408T101639
[2023-04-08T10:16:39.280+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-08T10:16:39.682+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
